smdx/smdx_00016.xml

```diff
@@ -22,7 +22,7 @@
         <symbol id="FULL_DUPLEX">1</symbol>
         <symbol id="FORCE_10MB">2</symbol>
         <symbol id="FORCE_100MB">3</symbol>
-        <symbol id="FORCE_1GB">3</symbol>
+        <symbol id="FORCE_1GB">4</symbol>
```

Comment:
I ran into this bug last week. So this is still open.
I've put up a fresh pull request for just this change.
https://github.com/sunspec/models/pull/58